src/property-interpolation.js

```diff
@@ -90,10 +90,13 @@
   };
 
   function propertyInterpolation(property, left, right) {
+    var ucProperty = property.replace(/-(.)/g, function(_, c) {
+      return c.toUpperCase();
+    });
+    if (ucProperty !== property) {
+      shared.deprecated('Hyphenated property names', '2016-03-22', 'Use camelCase instead.', true);
```

Comment:
I think you want to use `isDeprecated` since we shouldn't throw here after the deprecation period.

Also, since we need to support this feature for 3 months (including where it is removed from blink), we should convert hyphenated property names to camel case when `isDeprecated` returns `false`.